src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1963,8 +1961,6 @@
   "block.minecolonies.blockpostbox": "Postbox",
   "block.minecolonies.blockstash": "Stash",
   "block.minecolonies.blockstash.name": "Stash",
-  "block.minecolonies.blocksubstitution": "Placeholder Block",
-  "block.minecolonies.blocksubstitution.name": "Placeholder Block",
```

Comment:
This doesn't seem intended.

Reply:
Those don't exist in minecolonies anymore, were moved to structurize a long time ago.